Source/Core/Common/Network.cpp

```diff
@@ -43,7 +41,7 @@ std::string MacAddressToString(const MACAddress& mac)
                      mac[4], mac[5]);
 }
 
-std::optional<MACAddress> StringToMacAddress(const std::string& mac_string)
+std::optional<MACAddress> StringToMacAddress(std::string_view mac_string)
 {
   if (mac_string.empty())
     return {};
```

Comment:
You yourself recommended `std::nullopt` a while back, would be a good fit there. Since this PR only touches a single function, I see no reason not to go for it together with `string_view` change.